doc/api/cli.md

```diff
@@ -458,7 +458,7 @@ Node options that are allowed are:
 
 V8 options that are allowed are:
 - `--abort-on-uncaught-exception`
-- `--max_old_space_size`
+- `--max-old-space-size`
```

Comment:
Does not the previous variant hint on variations? If we unify here, should not we document that `_` is also acceptable?